src/cryptonote_core/tx_pool.cpp

```diff
@@ -680,9 +692,7 @@ namespace cryptonote
       bl.tx_hashes.push_back(tx_it->first);
       total_size += tx_it->second.blob_size;
       fee += tx_it->second.fee;
-#if 0
       best_coinbase = coinbase;
```

Comment:
If the check on line 669 fails, then `coinbase` is uninitialised. If I am reading this correctly, this should be happening until version 5 hits.